client/assets/style.scss

```diff
@@ -1,13 +1,44 @@
-@import "tools/index";
-@import "base/index";
-@import "pages/index";
-@import "./toastr.scss";
+@import 'tools/index';
+@import 'base/index';
+@import 'pages/index';
+@import './toastr.scss';
 
-@import url("fonts.scss");
+@import url('fonts.scss');
 
 // google font
 @import url('https://fonts.googleapis.com/css?family=Spectral+SC');
 @import url('https://fonts.googleapis.com/css?family=Ubuntu');
+// Import Abel
+@import url('https://fonts.googleapis.com/css?family=Abel');
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+h1 {
+  font-family: Abel;
```

Comment:
### Nitpick

Since the font-family has been defined in the body tag, all child elements auto-inherit from it.